chained_swapper/tests/test_chained_swapper.py

```diff
@@ -10,11 +10,11 @@
 from ..hooks import uninstall_hook
 
 
-class TestChainedSwapper(common.SavepointCase):
+class TestChainedSwapper(common.TransactionCase):
     @classmethod
     def setUpClass(cls):
         super(TestChainedSwapper, cls).setUpClass()
-        cls.env["res.lang"].load_lang("es_ES")
+        cls.env["res.lang"]._activate_lang("es_ES")
```

Comment:
You can directly remove this line, as English is always preloaded.

Reply:
IMO, this is necessary to test what the tests do.

Reply:
Oh, sorry, I saw badly that it loads English.